### DISCUSSION TITLE
Apple Airpods are connected but does not output sound (Linux Ubuntu 22.04)

### DISCUSSION THREAD

**Discussion by caseNpoint** [blueman-project/blueman#2209] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

Hope many of us who use the apple max pro wireless on the Linux Ubuntu can get some resolve. I have tried many solutions. I have deleted pulseAudio and installed pipeWire instead but still no resolve. Please assist if you have time. 
thank you much

**Comment by StrongTheDev** [comment]
this issue happens to my headsets as well (not apple). They connect but the sound keeps coming from the PC speaker

**Comment by infirit** [comment]
If it connects then blueman did it's job as the actual audio part is solely handled by PulseAudio or pipewire. Does it work properly if you use `bluetoothctl` from a terminal? If so then provide a blueman-applet log as described in rtoubleshooting https://github.com/blueman-project/blueman/wiki/Troubleshooting

**Reply by StrongTheDev** [reply]
oh yes i think thats the thing, because i did an upgrade and i recall it had to choose between pipewire and Pulseaudio and when it took pulseAudio, the headsets worked fine
